dlk/python/dlk/core/operators.py

```diff
@@ -3109,3 +3109,55 @@ def _check_consistency(self) -> None:
     @property
     def is_monotonic(self) -> bool:
         return False
+
+
+class Mean(Operator):
+    """Reduce mean operator.
+    Computes the mean of elements across dimensions of an input array.
+    This operator is referred as tf.reduce_mean, which is equivalent to np.mean.
+
+    Input
+    -----
+    A
+        Input: the array to reduce.
+    B
+        Axis: The dimensions to reduce.
+    Output
+    ------
+    C
+        The reduced output array
+    """
+
+    _input_names = ['A', 'B']
```

Comment:
how about using variable names `input` and `axis` insteaf of `A` and `B`?

Reply:
variable name changed in commit eb3a3cd